rust/worker/build.rs

```diff
@@ -17,6 +17,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .flag("-DHAVE_CXX0X")
         .flag("-fpic")
         .flag("-ftree-vectorize")
+        // .flag("-w") // Uncomment to suppress c++ warnings.
```

Comment:
This seems worth having. Happy to remove.

Reply:
Let's leave it out for now - I prefer the warnings cause pain until we have the correct solution to deal with them!